magento/magento2
# Minimum Qty Allowed in Shopping Cart not working on related product

### Description (*)

Solve **Minimum Qty Allowed in Shopping Cart** not working on related products

### Manual testing scenarios (*)

1. Let's take two simple products **Product A** and **Product B**
2. Set Minimum Qty Allowed in Shopping Cart = 10 in **Product A** from
Advanced Inventor->Minimum Qty Allowed in Shopping Cart
3. Assign **Product A** as a related product in **Product B**
3. Open PDP page of **Product B**
4. Check Product A checkbox from Related Products block on PDP
5. Add to cart **Product B** 

### Expected result (*)

Add to cart **Product A** with Qty = 10

### Actual result (*)

1. Add only 1 qty of Product A to the cart
2. Showing error "We don't have as many of some products as you want."

### Contribution checklist (*)
